BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -175,6 +199,28 @@ extension StackElement {
 /// A layout implementation that linearly lays out an array of children along either the horizontal or vertical axis.
 public struct StackLayout: Layout {
 
+    /// Default layout for a `Column`.
+    public static let defaultColumn: Self = StackLayout(
+        axis: .vertical,
+        alignment: Column.ColumnAlignment.leading.stackAlignment
+    )
+
+    /// Default layout for a `Row`.
+    public static let defaultRow: Self = StackLayout(
+        axis: .horizontal,
+        alignment: Row.RowAlignment.top.stackAlignment
+    )
+
+    /// Converts alignment to `RowAlignment`.
+    public var rowAlignment: Row.RowAlignment {
+        .init(alignment)
+    }
+
+    /// Converts alignment to `ColumnAlignment`.
+    public var columnAlignment: Column.ColumnAlignment {
+        .init(alignment)
+    }
```

Comment:
Should we move these into extensions defined at the bottom of `Row.Swift`/`Column.swift`?

Reply:
I have no opinion on this so I'll go ahead and move them 👍

Reply:
+1 agree with Westin; thanks!